src/runtime_src/xdp/profile/plugin/vp_base/utility.cpp

```diff
@@ -65,14 +65,20 @@ namespace xdp {
     xrt_core::get_xrt_info(pt) ;
 
     try {
-      pt.get<std::string>("build.zocl") ;
-    } catch (...) {
-      // We didn't find the zocl driver version, so we must not be on edge
-      return false ;
+      for (boost::property_tree::ptree::value_type& info : pt.get_child("drivers")) {
```

Comment:
should you use lambda function to do the search which would help shrink this code?

Reply:
I can add the suggested code improvement in future PR. The functionality in this PR is necessary for tomorrow's AIE bash.